resources/extruders/Mark2_extruder2.def.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": "Mark2_extruder2",
+    "version": 2,
+    "name": "Extruder 2",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "Mark2_for_Ultimaker2",
+        "position": "1"
+    },
+
+    "overrides": {
+        "extruder_nr": {
+            "default_value": 1,
+            "maximum_value": "1"
+        },
+        "machine_nozzle_offset_x": { "default_value": 0.0 },
```

Comment:
Please check if your Extruder 2 and Extruder 1 are at the same location. They have the same offsets (0, 0) right now.

Reply:
It's intentionally defined this way (it's a tool changer after all). Any offsets are (exclusively) processed by the firmware and must not depend on slicer settings (or on a specific slicer).